xcode/Subconscious/Shared/Library/UserDefaultsProperty.swift

```diff
@@ -27,14 +32,23 @@ import Foundation
     ) {
         self.key = key
         self.default = wrappedValue
+        self.subject = CurrentValueSubject(wrappedValue)
     }
     
+    // Projected value is accessable via the `$` notation.
+    var projectedValue: AnyPublisher<Value, Never> {
+        // removeDuplicates prevents us from firing a send for changes that
+        // did not actually update the equatable value.
+        self.subject.removeDuplicates().eraseToAnyPublisher()
+    }
+
     var wrappedValue: Value {
         get {
             scope.value(forKey: key) as? Value ?? `default`
         }
-        set {
+        nonmutating set {
```

Comment:
TIL this keyword exist

Reply:
Yeah another Swift oddity. This lets us use setter syntax for let-bound variables that point to this property.